### DISCUSSION TITLE
Import question

### DISCUSSION THREAD

**Discussion by jackmcd101** [rainbow-me/rainbowkit#629] [Q&A]
Hello! I’m getting ready to use RainbowKit and had a question: would the attached line of code be copy/pasted into an wallet section of a code base that might look like the attached? Or should it be pasted to an API section? 
![image](https://user-images.githubusercontent.com/59311318/182205923-3eb9b4db-a891-43f2-853e-a25eea9a7c3b.png)

![E8D00673-14D0-48BF-8392-331DC6B69157](https://user-images.githubusercontent.com/59311318/182189986-f3637374-33c8-41bf-9ce3-bad06f596f22.png)

**Comment by markdalgleish** [comment]
Hey! 👋

It looks like you're trying to drop a RainbowKit script tag into your page—is that right? At the moment RainbowKit only works with JS bundling tools ([vite](https://vitejs.dev), [esbuild](https://esbuild.github.io), [webpack](https://webpack.js.org) etc.) + React, with most developers reaching for a framework like [Next.js](https://nextjs.org) to manage this for them.
